apps/webapp/src/components/staking/validator-info-row.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  BondingState_BondingStateEnum,
+  ValidatorInfo,
+  ValidatorState_ValidatorStateEnum,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/stake/v1/stake_pb';
+import {
+  getValidator,
+  joinLoHiAmount,
+  getValidatorRewardRateFromValidatorInfoOptional,
+  getStateEnumFromValidatorInfo,
+  getBondingStateEnumFromValidatorInfo,
+} from '@penumbra-zone/types';
+import { TableRow, TableCell } from '@penumbra-zone/ui';
+import { Oval } from 'react-loader-spinner';
+
+const getStateLabel = (validatorInfo: ValidatorInfo): string =>
+  ValidatorState_ValidatorStateEnum[getStateEnumFromValidatorInfo(validatorInfo)];
+
+const getBondingStateLabel = (validatorInfo: ValidatorInfo): string =>
+  BondingState_BondingStateEnum[getBondingStateEnumFromValidatorInfo(validatorInfo)];
+
+export const ValidatorInfoRow = ({
+  loading,
+  validatorInfo,
+  votingPowerByValidatorInfo,
+}: {
+  loading: boolean;
+  validatorInfo: ValidatorInfo;
+  votingPowerByValidatorInfo: Map<ValidatorInfo, number>;
+}) => {
+  const rewardRate = getValidatorRewardRateFromValidatorInfoOptional(validatorInfo);
+  const commission = rewardRate ? Number(joinLoHiAmount(rewardRate)) : '';
```

Comment:
Not sure how to calculate commission. At the moment, I'm just `Number`ifying the reward rate. Reviewers, how is commission calculated?